cmake/Toolchain-aarch64.cmake

```diff
@@ -0,0 +1,36 @@
+set(CMAKE_SYSTEM_NAME "Generic" CACHE STRING "")
```

Comment:
Should this file be renamed to include the fact that it is xilinx and bare metal? If I want to compile acados natively on linux on my ARM 64-bit board I should not use this file.

Reply:
How about Toolchain-xilinx-a53-bare.cmake?
-->you can use this toolchain for every xilinx SoC-based board that has the a53 core if you want bare-metal compilation.